src/declarations/stencil-private.ts

```diff
@@ -346,6 +356,13 @@ export interface BundleEntryInputs {
   [entryKey: string]: string;
 }
 
+/**
+ * A note regarding Rollup types:
```

Comment:
@rwaskiewicz do you consider this Technical debt? Would we have a ticket in the backlog already if so?

Reply:
I don't think so, it's a pretty deep archtectural aspect of the system (I think, in so far as I can tell rn). I'll add something to the backlog as a low priority spike

Reply:
STENCIL-175